build.gradle

```diff
@@ -36,7 +36,8 @@ buildscript {
         ktxVersion = '1.0.1'
         lifecycleVersion = '2.1.0-alpha02'
         materialVersion = '1.0.0'
-        navigationVersion = '1.0.0-rc02'
+        navigationVersion = '2.0.0-rc02'
+        navigationSafeArgsVersion = '1.0.0-rc02'
```

Comment:
Navigation and Safe Args should always align in version, so I think it makes sense to collapse these into just a single version number.

Reply:
Done